src/wp-includes/rest-api.php

```diff
@@ -1691,9 +1692,16 @@ function rest_validate_value_from_schema( $value, $args, $param = '' ) {
 		}
 	}
 
-	if ( in_array( $args['type'], array( 'integer', 'number' ), true ) && ! is_numeric( $value ) ) {
-		/* translators: 1: Parameter, 2: Type name. */
-		return new WP_Error( 'rest_invalid_param', sprintf( __( '%1$s is not of type %2$s.' ), $param, $args['type'] ) );
+	if ( in_array( $args['type'], array( 'integer', 'number' ), true ) ) {
+		if ( ! is_numeric( $value ) ) {
+			/* translators: 1: Parameter, 2: Type name. */
+			return new WP_Error( 'rest_invalid_param', sprintf( __( '%1$s is not of type %2$s.' ), $param, $args['type'] ) );
+		}
+
+		if ( isset( $args['multipleOf'] ) && ! rest_is_integer( $value / $args['multipleOf'] ) ) {
```

Comment:
Can we use the modules operator instead?

Reply:
The modulo operator can't return decimal remainder, so I'll replace division with `fmod`.